2024-10-24T13:03:56.280347009Z I1024 13:03:56.280185    7167 start.go:68] Version: machine-config-daemon-4.6.0-202006240615.p0-3074-g54144b31-dirty (54144b315275a22cc9140677792ef5ed0189bcde)
2024-10-24T13:03:56.281617709Z I1024 13:03:56.281548    7167 update.go:2708] Running: mount --rbind /run/secrets /rootfs/run/secrets
2024-10-24T13:03:56.285676419Z I1024 13:03:56.285602    7167 update.go:2708] Running: mount --rbind /usr/bin /rootfs/run/machine-config-daemon-bin
2024-10-24T13:03:56.290605739Z I1024 13:03:56.290452    7167 daemon.go:517] using appropriate binary for source=rhel-9 target=rhel-9
2024-10-24T13:03:56.461559572Z I1024 13:03:56.461455    7167 daemon.go:570] Invoking re-exec /run/bin/machine-config-daemon
2024-10-24T13:03:56.500552860Z I1024 13:03:56.500425    7167 start.go:68] Version: machine-config-daemon-4.6.0-202006240615.p0-3074-g54144b31-dirty (54144b315275a22cc9140677792ef5ed0189bcde)
2024-10-24T13:03:56.500994041Z I1024 13:03:56.500895    7167 update.go:2708] Running: systemctl daemon-reload
2024-10-24T13:03:57.093486247Z I1024 13:03:57.093425    7167 rpm-ostree.go:56] Enabled workaround for bug 2111817
2024-10-24T13:03:57.093648207Z E1024 13:03:57.093618    7167 image_manager_helper.go:133] Merged secret file does not exist; defaulting to cluster pull secret
2024-10-24T13:03:57.093716737Z I1024 13:03:57.093661    7167 image_manager_helper.go:194] Linking rpm-ostree authfile to /var/lib/kubelet/config.json
2024-10-24T13:03:57.462176963Z I1024 13:03:57.462075    7167 daemon.go:320] Booted osImageURL: registry.build02.ci.openshift.org/ci-op-2fcpj5j6/stable@sha256:0943a4e3eb4b3c1cb77764182381d3ed03c0ff8d5f1992722081c06828915232 (418.94.202410230039-0) cae3f8f7a82712b3758e60f814dd9b603402853c706cd3762a85503937ba25fa
2024-10-24T13:03:57.463712733Z I1024 13:03:57.463674    7167 start.go:134] overriding kubernetes api to https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443
2024-10-24T13:03:57.464757883Z I1024 13:03:57.464730    7167 metrics.go:92] Registering Prometheus metrics
2024-10-24T13:03:57.464912073Z I1024 13:03:57.464896    7167 metrics.go:99] Starting metrics listener on 127.0.0.1:8797
2024-10-24T13:03:57.526311950Z I1024 13:03:57.526254    7167 simple_featuregate_reader.go:171] Starting feature-gate-detector
2024-10-24T13:03:57.526956390Z I1024 13:03:57.526928    7167 writer.go:88] NodeWriter initialized with credentials from /var/lib/kubelet/kubeconfig
2024-10-24T13:03:57.533316700Z I1024 13:03:57.533248    7167 start.go:214] FeatureGates initialized: knownFeatureGates=[AWSEFSDriverVolumeMetrics AdminNetworkPolicy AlibabaPlatform AzureWorkloadIdentity BareMetalLoadBalancer BuildCSIVolumes ChunkSizeMiB CloudDualStackNodeIPs DisableKubeletCloudCredentialProviders GCPLabelsTags HardwareSpeed IngressControllerLBSubnetsAWS KMSv1 ManagedBootImages MetricsServer MultiArchInstallAWS MultiArchInstallGCP NetworkDiagnosticsConfig NetworkLiveMigration NodeDisruptionPolicy OpenShiftPodSecurityAdmission PrivateHostedZoneAWS SetEIPForNLBIngressController VSphereControlPlaneMachineSet VSphereDriverConfiguration VSphereStaticIPs ValidatingAdmissionPolicy AWSClusterHostedDNS AdditionalRoutingCapabilities AutomatedEtcdBackup BootcNodeManagement CSIDriverSharedResource ClusterAPIInstall ClusterAPIInstallIBMCloud ClusterMonitoringConfig DNSNameResolver DynamicResourceAllocation EtcdBackendQuota EventedPLEG Example ExternalOIDC GCPClusterHostedDNS GatewayAPI ImageStreamImportMode IngressControllerDynamicConfigurationManager InsightsConfig InsightsConfigAPI InsightsOnDemandDataGather InsightsRuntimeExtractor MachineAPIMigration MachineAPIOperatorDisableMachineHealthCheckController MachineAPIProviderOpenStack MachineConfigNodes ManagedBootImagesAWS MaxUnavailableStatefulSet MetricsCollectionProfiles MixedCPUsAllocation MultiArchInstallAzure NetworkSegmentation NewOLM NodeSwap OVNObservability OnClusterBuild PersistentIPsForVirtualization PinnedImages PlatformOperators ProcMountType RouteAdvertisements RouteExternalCertificate ServiceAccountTokenNodeBinding SignatureStores SigstoreImageVerification TranslateStreamCloseWebsocketRequests UpgradeStatus UserNamespacesPodSecurityStandards UserNamespacesSupport VSphereMultiNetworks VSphereMultiVCenters VolumeGroupSnapshot]
2024-10-24T13:03:57.533316700Z I1024 13:03:57.533296    7167 update.go:2724] "Starting to manage node: ci-op-2fcpj5j6-f6035-2lklf-master-0"
2024-10-24T13:03:57.533919250Z I1024 13:03:57.533855    7167 event.go:377] Event(v1.ObjectReference{Kind:"Node", Namespace:"openshift-machine-config-operator", Name:"ci-op-2fcpj5j6-f6035-2lklf-master-0", UID:"70cf08f3-a1d7-4bf2-aa24-f4dca7982067", APIVersion:"v1", ResourceVersion:"", FieldPath:""}): type: 'Normal' reason: 'FeatureGatesInitialized' FeatureGates updated to featuregates.Features{Enabled:[]v1.FeatureGateName{"AWSEFSDriverVolumeMetrics", "AdminNetworkPolicy", "AlibabaPlatform", "AzureWorkloadIdentity", "BareMetalLoadBalancer", "BuildCSIVolumes", "ChunkSizeMiB", "CloudDualStackNodeIPs", "DisableKubeletCloudCredentialProviders", "GCPLabelsTags", "HardwareSpeed", "IngressControllerLBSubnetsAWS", "KMSv1", "ManagedBootImages", "MetricsServer", "MultiArchInstallAWS", "MultiArchInstallGCP", "NetworkDiagnosticsConfig", "NetworkLiveMigration", "NodeDisruptionPolicy", "OpenShiftPodSecurityAdmission", "PrivateHostedZoneAWS", "SetEIPForNLBIngressController", "VSphereControlPlaneMachineSet", "VSphereDriverConfiguration", "VSphereStaticIPs", "ValidatingAdmissionPolicy"}, Disabled:[]v1.FeatureGateName{"AWSClusterHostedDNS", "AdditionalRoutingCapabilities", "AutomatedEtcdBackup", "BootcNodeManagement", "CSIDriverSharedResource", "ClusterAPIInstall", "ClusterAPIInstallIBMCloud", "ClusterMonitoringConfig", "DNSNameResolver", "DynamicResourceAllocation", "EtcdBackendQuota", "EventedPLEG", "Example", "ExternalOIDC", "GCPClusterHostedDNS", "GatewayAPI", "ImageStreamImportMode", "IngressControllerDynamicConfigurationManager", "InsightsConfig", "InsightsConfigAPI", "InsightsOnDemandDataGather", "InsightsRuntimeExtractor", "MachineAPIMigration", "MachineAPIOperatorDisableMachineHealthCheckController", "MachineAPIProviderOpenStack", "MachineConfigNodes", "ManagedBootImagesAWS", "MaxUnavailableStatefulSet", "MetricsCollectionProfiles", "MixedCPUsAllocation", "MultiArchInstallAzure", "NetworkSegmentation", "NewOLM", "NodeSwap", "OVNObservability", "OnClusterBuild", "PersistentIPsForVirtualization", "PinnedImages", "PlatformOperators", "ProcMountType", "RouteAdvertisements", "RouteExternalCertificate", "ServiceAccountTokenNodeBinding", "SignatureStores", "SigstoreImageVerification", "TranslateStreamCloseWebsocketRequests", "UpgradeStatus", "UserNamespacesPodSecurityStandards", "UserNamespacesSupport", "VSphereMultiNetworks", "VSphereMultiVCenters", "VolumeGroupSnapshot"}}
2024-10-24T13:03:57.540693470Z I1024 13:03:57.540640    7167 image_manager_helper.go:92] Running captured: rpm-ostree status
2024-10-24T13:03:57.602246567Z I1024 13:03:57.601816    7167 daemon.go:1761] State: idle
2024-10-24T13:03:57.602246567Z Deployments:
2024-10-24T13:03:57.602246567Z * ostree-unverified-registry:registry.build02.ci.openshift.org/ci-op-2fcpj5j6/stable@sha256:0943a4e3eb4b3c1cb77764182381d3ed03c0ff8d5f1992722081c06828915232
2024-10-24T13:03:57.602246567Z                    Digest: sha256:0943a4e3eb4b3c1cb77764182381d3ed03c0ff8d5f1992722081c06828915232
2024-10-24T13:03:57.602246567Z                   Version: 418.94.202410230039-0 (2024-10-23T00:43:29Z)
2024-10-24T13:03:57.602246567Z 
2024-10-24T13:03:57.602246567Z   ostree-image-signed:oci-archive:/rhcos-418.94.202410090804-0-ostree.x86_64.ociarchive
2024-10-24T13:03:57.602246567Z                    Digest: sha256:e404c1b154fd43dcb10b3609f6b608bbd2c61bbab96ba1f737a2f2f3174131d1
2024-10-24T13:03:57.602246567Z                   Version: 418.94.202410090804-0 (2024-10-09T08:08:35Z)
2024-10-24T13:03:57.603411887Z I1024 13:03:57.603371    7167 coreos.go:53] CoreOS aleph version: mtime=2022-08-01 23:42:11 +0000 UTC
2024-10-24T13:03:57.603411887Z {
2024-10-24T13:03:57.603411887Z    "container-image": {
2024-10-24T13:03:57.603411887Z       "image-digest": "sha256:a8e53889f43e302481b2f5efcf497bc62fba75c415d5fde1a127ce42088c33bf",
2024-10-24T13:03:57.603411887Z       "image-labels": {
2024-10-24T13:03:57.603411887Z          "containers.bootc": "1",
2024-10-24T13:03:57.603411887Z          "coreos-assembler.image-config-checksum": "626c78cc9e2da6ffd642d678ac6109f5532e817e107932464c8222f8d3492491",
2024-10-24T13:03:57.603411887Z          "coreos-assembler.image-input-checksum": "0236dfae55fc000b77b96858150e5919492d52aa8cbaa5cf1ef53bfd2e88796f",
2024-10-24T13:03:57.603411887Z          "io.openshift.build.version-display-names": "machine-os=Red Hat Enterprise Linux CoreOS",
2024-10-24T13:03:57.603411887Z          "io.openshift.build.versions": "machine-os=418.94.202410090804-0",
2024-10-24T13:03:57.603411887Z          "org.opencontainers.image.revision": "b753d25572e525e1a2df8bfe412b60a1dbf39674",
2024-10-24T13:03:57.603411887Z          "org.opencontainers.image.source": "https://github.com/openshift/os",
2024-10-24T13:03:57.603411887Z          "org.opencontainers.image.version": "418.94.202410090804-0",
2024-10-24T13:03:57.603411887Z          "ostree.bootable": "true",
2024-10-24T13:03:57.603411887Z          "ostree.commit": "8e43827c9d7336ac150ed810d856a0e47543f9599259a0e119283e67d5abd74a",
2024-10-24T13:03:57.603411887Z          "ostree.final-diffid": "sha256:12787d84fa137cd5649a9005efe98ec9d05ea46245fdc50aecb7dd007f2035b1",
2024-10-24T13:03:57.603411887Z          "ostree.linux": "5.14.0-427.40.1.el9_4.x86_64",
2024-10-24T13:03:57.603411887Z          "rpmostree.inputhash": "55fb4822fa6e440ded69bb749d02bd2b6cd97bea402df04194ed7d3b9bd79d09"
2024-10-24T13:03:57.603411887Z       },
2024-10-24T13:03:57.603411887Z       "image-name": "oci-archive:/rhcos-418.94.202410090804-0-ostree.x86_64.ociarchive"
2024-10-24T13:03:57.603411887Z    },
2024-10-24T13:03:57.603411887Z    "osbuild-version": "130",
2024-10-24T13:03:57.603411887Z    "ostree-commit": "8e43827c9d7336ac150ed810d856a0e47543f9599259a0e119283e67d5abd74a",
2024-10-24T13:03:57.603411887Z    "ref": "docker://ostree-image-signed:oci-archive:/rhcos-418.94.202410090804-0-ostree.x86_64.ociarchive",
2024-10-24T13:03:57.603411887Z    "version": "418.94.202410090804-0"
2024-10-24T13:03:57.603411887Z }
2024-10-24T13:03:57.603709957Z I1024 13:03:57.603658    7167 coreos.go:70] Ignition provisioning: time=2024-10-24T12:58:15Z
2024-10-24T13:03:57.603771387Z I1024 13:03:57.603757    7167 image_manager_helper.go:92] Running captured: journalctl --list-boots
2024-10-24T13:03:57.611284187Z I1024 13:03:57.611239    7167 daemon.go:1770] journalctl --list-boots:
2024-10-24T13:03:57.611284187Z IDX BOOT ID                          FIRST ENTRY                 LAST ENTRY
2024-10-24T13:03:57.611284187Z  -1 c557d4c3c090447a8e84d6a92134fb47 Thu 2024-10-24 12:55:21 UTC Thu 2024-10-24 12:59:36 UTC
2024-10-24T13:03:57.611284187Z   0 818b54fbeb8f48af935307bb3facd639 Thu 2024-10-24 12:59:55 UTC Thu 2024-10-24 13:03:57 UTC
2024-10-24T13:03:57.611360357Z I1024 13:03:57.611343    7167 image_manager_helper.go:92] Running captured: systemctl list-units --state=failed --no-legend
2024-10-24T13:03:57.623152817Z I1024 13:03:57.623091    7167 daemon.go:1786] systemd service state: OK
2024-10-24T13:03:57.623283866Z I1024 13:03:57.623268    7167 daemon.go:1358] Starting MachineConfigDaemon
2024-10-24T13:03:57.623391557Z I1024 13:03:57.623378    7167 daemon.go:1365] Enabling Kubelet Healthz Monitor
2024-10-24T13:03:57.679765604Z I1024 13:03:57.679724    7167 daemon.go:2842] Found 0 requested local packages in the first deployment
2024-10-24T13:03:58.536361781Z I1024 13:03:58.536311    7167 daemon.go:651] Node ci-op-2fcpj5j6-f6035-2lklf-master-0 is part of the control plane
2024-10-24T13:03:58.655763056Z E1024 13:03:58.655702    7167 writer.go:226] Marking Degraded due to: could not get ControllerConfig: controllerconfig.machineconfiguration.openshift.io "machine-config-controller" not found
2024-10-24T13:04:00.673339577Z I1024 13:04:00.673277    7167 daemon.go:739] Transitioned from state:  -> Degraded
2024-10-24T13:04:00.673339577Z I1024 13:04:00.673312    7167 daemon.go:742] Transitioned from degraded/unreconcilable reason  -> could not get ControllerConfig: controllerconfig.machineconfiguration.openshift.io "machine-config-controller" not found
2024-10-24T13:04:00.673406068Z E1024 13:04:00.673360    7167 writer.go:226] Marking Degraded due to: could not get ControllerConfig: controllerconfig.machineconfiguration.openshift.io "machine-config-controller" not found
2024-10-24T13:04:01.442160128Z E1024 13:04:01.442087    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:04:01.458831607Z I1024 13:04:01.458764    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 8929
2024-10-24T13:04:01.467585407Z E1024 13:04:01.467539    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:04:01.480109416Z I1024 13:04:01.480054    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 8931
2024-10-24T13:04:01.491552616Z E1024 13:04:01.491476    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:04:01.507894735Z I1024 13:04:01.507846    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 8941
2024-10-24T13:04:02.687483579Z E1024 13:04:02.687373    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:04:02.699012519Z I1024 13:04:02.698971    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 8978
2024-10-24T13:04:08.678251315Z I1024 13:04:08.678162    7167 daemon.go:1935] Running: /run/machine-config-daemon-bin/nmstatectl persist-nic-names --root / --kargs-out /tmp/nmstate-kargs3159287302 --cleanup
2024-10-24T13:04:08.707656884Z [2024-10-24T13:04:08Z INFO  nmstatectl] Nmstate version: 2.2.33
2024-10-24T13:04:08.710784294Z 
2024-10-24T13:04:08.710827444Z [2024-10-24T13:04:08Z INFO  nmstatectl::persist_nic] /etc/systemd/network does not exist, no need to clean up
2024-10-24T13:04:08.711384664Z I1024 13:04:08.711220    7167 node.go:23] No machineconfiguration.openshift.io/currentConfig annotation on node ci-op-2fcpj5j6-f6035-2lklf-master-0: map[csi.volume.kubernetes.io/nodeid:{"pd.csi.storage.gke.io":"projects/XXXXXXXXXXXXXXXXXXXXXXXX/zones/us-central1-a/instances/ci-op-2fcpj5j6-f6035-2lklf-master-0"} k8s.ovn.org/host-cidrs:["10.0.0.3/32"] k8s.ovn.org/l3-gateway-config:{"default":{"mode":"shared","bridge-id":"br-ex","interface-id":"br-ex_ci-op-2fcpj5j6-f6035-2lklf-master-0","mac-address":"42:01:0a:00:00:03","ip-addresses":["10.0.0.3/32"],"ip-address":"10.0.0.3/32","next-hops":["10.0.0.1"],"next-hop":"10.0.0.1","node-port-enable":"true","vlan-id":"0"}} k8s.ovn.org/network-ids:{"default":"0"} k8s.ovn.org/node-chassis-id:7e3fc786-0823-44c4-b12c-40377bfc1950 k8s.ovn.org/node-gateway-router-lrp-ifaddrs:{"default":{"ipv4":"100.64.0.3/16"}} k8s.ovn.org/node-id:3 k8s.ovn.org/node-masquerade-subnet:{"ipv4":"169.254.0.0/17","ipv6":"fd69::/112"} k8s.ovn.org/node-mgmt-port-mac-addresses:{"default":"fa:77:c2:a4:20:7a"} k8s.ovn.org/node-primary-ifaddr:{"ipv4":"10.0.0.3/32"} k8s.ovn.org/node-subnets:{"default":["10.129.0.0/23"]} k8s.ovn.org/node-transit-switch-port-ifaddr:{"ipv4":"100.88.0.3/16"} k8s.ovn.org/remote-zone-migrated:ci-op-2fcpj5j6-f6035-2lklf-master-0 k8s.ovn.org/zone-name:ci-op-2fcpj5j6-f6035-2lklf-master-0 machineconfiguration.openshift.io/lastObservedServerCAAnnotation:false machineconfiguration.openshift.io/lastSyncedControllerConfigResourceVersion:8978 machineconfiguration.openshift.io/reason:could not get ControllerConfig: controllerconfig.machineconfiguration.openshift.io "machine-config-controller" not found machineconfiguration.openshift.io/state:Degraded volumes.kubernetes.io/controller-managed-attach-detach:true], in cluster bootstrap, loading initial node annotation from /etc/machine-config-daemon/node-annotations.json
2024-10-24T13:04:08.712349474Z I1024 13:04:08.712302    7167 node.go:52] Setting initial node config: rendered-master-1f75404f08afc3926de8a846ea4bc6ff
2024-10-24T13:04:08.725942073Z I1024 13:04:08.725862    7167 daemon.go:1659] In bootstrap mode
2024-10-24T13:04:08.733173033Z I1024 13:04:08.733032    7167 daemon.go:1597] Previous boot ostree-finalize-staged.service appears successful
2024-10-24T13:04:08.733173033Z I1024 13:04:08.733065    7167 daemon.go:1714] Current+desired config: rendered-master-1f75404f08afc3926de8a846ea4bc6ff
2024-10-24T13:04:08.733173033Z I1024 13:04:08.733072    7167 daemon.go:1729] state: Done
2024-10-24T13:04:08.733173033Z I1024 13:04:08.733103    7167 update.go:2708] Running: rpm-ostree cleanup -r
2024-10-24T13:04:14.853743955Z Bootloader updated; bootconfig swap: yes; bootversion: boot.1.1, deployment count change: -1
2024-10-24T13:04:18.537482801Z Pruned images: 1 (layers: 29)
2024-10-24T13:04:22.134556621Z Freed: 864.4?MB (pkgcache branches: 0)
2024-10-24T13:04:22.756019846Z I1024 13:04:22.755969    7167 update.go:2724] "No bootstrap pivot required; unlinking bootstrap node annotations"
2024-10-24T13:04:22.759566647Z I1024 13:04:22.759440    7167 daemon.go:2133] Validating against current config rendered-master-1f75404f08afc3926de8a846ea4bc6ff
2024-10-24T13:04:22.759945626Z I1024 13:04:22.759851    7167 daemon.go:2022] SSH key location update required. Moving SSH keys from "/home/core/.ssh/authorized_keys" to "/home/core/.ssh/authorized_keys.d/ignition".
2024-10-24T13:04:22.769713076Z I1024 13:04:22.769612    7167 update.go:2406] updating SSH keys
2024-10-24T13:04:22.770080176Z I1024 13:04:22.769984    7167 update.go:2307] Writing SSH keys to "/home/core/.ssh/authorized_keys.d/ignition"
2024-10-24T13:04:22.770080176Z I1024 13:04:22.770033    7167 update.go:2272] Creating missing SSH key dir at "/home/core/.ssh/authorized_keys.d"
2024-10-24T13:04:26.485313700Z I1024 13:04:26.485231    7167 image_manager_helper.go:92] Running captured: rpm-ostree kargs
2024-10-24T13:04:26.553752468Z I1024 13:04:26.553177    7167 update.go:2724] "Validated on-disk state"
2024-10-24T13:04:26.569720547Z I1024 13:04:26.569644    7167 daemon.go:2235] Completing update to target MachineConfig: rendered-master-1f75404f08afc3926de8a846ea4bc6ff
2024-10-24T13:04:36.603851309Z I1024 13:04:36.603761    7167 update.go:2724] "Update completed for config rendered-master-1f75404f08afc3926de8a846ea4bc6ff and node has been successfully uncordoned"
2024-10-24T13:04:36.618710111Z I1024 13:04:36.618648    7167 daemon.go:2260] In desired state MachineConfig: rendered-master-1f75404f08afc3926de8a846ea4bc6ff
2024-10-24T13:04:36.634064262Z I1024 13:04:36.633971    7167 config_drift_monitor.go:246] Config Drift Monitor started
2024-10-24T13:04:39.573441962Z I1024 13:04:39.573356    7167 daemon.go:742] Transitioned from degraded/unreconcilable reason could not get ControllerConfig: controllerconfig.machineconfiguration.openshift.io "machine-config-controller" not found -> 
2024-10-24T13:04:58.913608996Z I1024 13:04:58.913547    7167 certificate_writer.go:185] Unable to decode cert into a pem block. Cert is either empty or invalid.
2024-10-24T13:04:58.913608996Z I1024 13:04:58.913598    7167 update.go:2724] "Cert not found in kubeconfig. This means we need to write to disk. Subject is: kube-apiserver-lb-signer"
2024-10-24T13:04:58.916936836Z I1024 13:04:58.916896    7167 update.go:2724] "Skipping kubelet restart"
2024-10-24T13:04:58.919700097Z I1024 13:04:58.919666    7167 update.go:2724] "Cert not found in kubeconfig. This means we need to write to disk. Subject is: kube-apiserver-localhost-signer"
2024-10-24T13:04:58.921933707Z I1024 13:04:58.921892    7167 update.go:2724] "Skipping kubelet restart"
2024-10-24T13:04:58.924800987Z I1024 13:04:58.924735    7167 update.go:2724] "Cert not found in kubeconfig. This means we need to write to disk. Subject is: openshift-kube-apiserver-operator_localhost-recovery-serving-signer@1729775011"
2024-10-24T13:04:58.927331227Z I1024 13:04:58.927272    7167 update.go:2724] "Skipping kubelet restart"
2024-10-24T13:04:58.930754397Z I1024 13:04:58.930661    7167 certificate_writer.go:233] Writing new Data to /etc/kubernetes/kubeconfig: clusters:
2024-10-24T13:04:58.930754397Z - cluster:
2024-10-24T13:04:58.930754397Z     certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURNakNDQWhxZ0F3SUJBZ0lJT2xlbEFaUklSSlF3RFFZSktvWklodmNOQVFFTEJRQXdOekVTTUJBR0ExVUUKQ3hNSmIzQmxibk5vYVdaME1TRXdId1lEVlFRREV4aHJkV0psTFdGd2FYTmxjblpsY2kxc1lpMXphV2R1WlhJdwpIaGNOTWpReE1ESTBNVEkwT1RNeVdoY05NelF4TURJeU1USTBPVE15V2pBM01SSXdFQVlEVlFRTEV3bHZjR1Z1CmMyaHBablF4SVRBZkJnTlZCQU1UR0d0MVltVXRZWEJwYzJWeWRtVnlMV3hpTFhOcFoyNWxjakNDQVNJd0RRWUoKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBT2NuTW5OcDhtYkp6UDBZUnRmUDJlUUxVYXhEUkNXaAp3TGpYUGVJK1JKTmM1WFJUZjhLN2NuRnZaTStWeGVEdnQzdnZIbUsrdjZlZVQydk00c1RlMUZpNHkzK04rNTJFCkFaZkFzeDVqWmtPTE1xWG1ZQ0V0Y00waUU1aENEbHFqK2lxejMrTjkvaDlyL3dsa0djdUxqRmF0dmZZUGUyYUQKb3JZM1MveFdjY1dwQm1QTk5iNmcrSmdOYkZrWnR3U2dQMnJBMERnSlFObStTRWxTVlhzT0Q0ejZOSmVRWVVHUwozUyszbXVML1IwdHFIVTZvcUpVVVM1aStTSTB6VXRHWFNQRjcxL1Nad3AwWnpscUpPbkYxK29Ld3ZMd1ZZZW5KCjFjaHpIMm9vc3dUeklsMkdicnJpUWlpRlhjTmF0dzVIUFlBVWovdTVteGMyVUZkcTBCUy9rUzhDQXdFQUFhTkMKTUVBd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCL3dRRk1BTUJBZjh3SFFZRFZSME9CQllFRkhsMgozOC9KWlB6KzBrL0pzZXlOR1NTbnhJV2JNQTBHQ1NxR1NJYjNEUUVCQ3dVQUE0SUJBUUN0L2Y5MXd2RWY1eVhCCk02aVZIeVlTNXF1VmtnS095Vy9WbE9pV0JyQzNuenVtemJ5S0JLQ0NEVkZWME9iRkJVSjk0dWhZR0NNMzJPTGUKa09GM2cvc05NdGpqK3o4T1dhdVFxUThnb2N6cEMrTkpiL0FOYUF0ZEZOK2VsNWdMeTEzc1hQc3U4L2dpZEFPYgpqYXBaN3drOFB4MzNuL3NRYnN2TTdlU3VXM21Xc0VXQ3puWE1BaVh6a3Q0UjJqTHhZc2xGY3F6QXlqcVlYT2VJCjhoWWNnbUxWazVSMlpUS3RnTjZLMEgxQlJJUE5QWjc2R0tqSzBLcGd1enlSSEpOOHk2aUQvT2U0Nzc5ejB0MWsKc0xPc1k4bXppMWJSSEVWMzFPZzdzNm5GeTQyRUNNNVdaYi9idHFkSm4ySy9BUUYzV0N0ZFg2VTkrclN1U2NPQgo1Zk9hdGdPYgotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCi0tLS0tQkVHSU4gQ0VSVElGSUNBVEUtLS0tLQpNSUlEUURDQ0FpaWdBd0lCQWdJSWF4cHVKNGI5bi93d0RRWUpLb1pJaHZjTkFRRUxCUUF3UGpFU01CQUdBMVVFCkN4TUpiM0JsYm5Ob2FXWjBNU2d3SmdZRFZRUURFeDlyZFdKbExXRndhWE5sY25abGNpMXNiMk5oYkdodmMzUXQKYzJsbmJtVnlNQjRYRFRJME1UQXlOREV5TkRrek1Wb1hEVE0wTVRBeU1qRXlORGt6TVZvd1BqRVNNQkFHQTFVRQpDeE1KYjNCbGJuTm9hV1owTVNnd0pnWURWUVFERXg5cmRXSmxMV0Z3YVhObGNuWmxjaTFzYjJOaGJHaHZjM1F0CmMybG5ibVZ5TUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUF2Q0xDNEg4YzlxUHgKaXFFZ3R5bjltYmxFTERJSkEvL2czQTZ2dWhWcHJtVnhGS2RSK1NMZE43aXVQWFdKeU5Kb011alJwTGg0TzhUeAp5bkhPSEFTSDNOWnUyME5CeFBOd003Y00yWU4zNXJZS013Z0dNZzZIL21GQ3lNblhvMndUYnBHMGtWYmsxSStECnBRRHVDQ2Q1TndhOE15Rk1SbXhDN0ZXU0QvTlh5Sk45NitUeXdQNjJkOVVCd3A4RjJUVjBJd0VZSlBwN1FNQksKVERlZis4bVdLYU1qWXBhVGJEVGgvY2U2QjZmYVNmbGlHaEJxc0tTV2ZVUEJzcy80bmxXc0RLT0xhZ29ESFFUdAp1Rll5cVh2ZUgzbGllS2QwZXUvbUNVaW9MenExR1E5b0lkTkhicVlmK3pXQjNoaUpQd2J4ZlJxMDErTkIrMUdhCkh0VDRxS2tMaVFJREFRQUJvMEl3UURBT0JnTlZIUThCQWY4RUJBTUNBcVF3RHdZRFZSMFRBUUgvQkFVd0F3RUIKL3pBZEJnTlZIUTRFRmdRVUFvWDBRTmIveWdtUHlGVm5XclBjLzNsS1MrNHdEUVlKS29aSWh2Y05BUUVMQlFBRApnZ0VCQUZuamF2OUpOQjZOU0gySGdPbGoybXZxWFNZT0ttTEQxYzZXbHVKWE9TaDJZT0U5aUdEWkpFcE5HZngzCnNYZGszUEtwOXZoZU1kZkJJV0RtZTJIN29jSmlQVWw3SGx6UDBQdlNsNWYvOUphZDNLRWw1a29KTXhhb2JITWcKODZCVDVMZzNUUTNIMzY5Qnc5eWVkWXV3Nldha2w4ekdnWXNIUmREaUJzUUZZaGVPYkhZY2JXc1ptc01NMzh1cQo4b1h1eXNZTTdMdm0xWWNlUzUwODNYMFcyMnlxbnpJR3NiY2xCNk1hTk8zR0dPYVVTTS9qUysxbmdzZTFudUZEClg3YW8rTjBpckxFVWR5OC95b0plNC9hVjl0SmxMcVBKKzZ0TlR2TEJVN0k2cElvOGxScWU3OWNNVVhrZmNkU0IKL2prZlNkVVFCQnYvcWR2Q25QM0R6MmROc1Q0PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCi0tLS0tQkVHSU4gQ0VSVElGSUNBVEUtLS0tLQpNSUlEVERDQ0FqU2dBd0lCQWdJSU5WaklXUUlpMUpJd0RRWUpLb1pJaHZjTkFRRUxCUUF3UkRFU01CQUdBMVVFCkN4TUpiM0JsYm5Ob2FXWjBNUzR3TEFZRFZRUURFeVZyZFdKbExXRndhWE5sY25abGNpMXpaWEoyYVdObExXNWwKZEhkdmNtc3RjMmxuYm1WeU1CNFhEVEkwTVRBeU5ERXlORGt6TVZvWERUTTBNVEF5TWpFeU5Ea3pNVm93UkRFUwpNQkFHQTFVRUN4TUpiM0JsYm5Ob2FXWjBNUzR3TEFZRFZRUURFeVZyZFdKbExXRndhWE5sY25abGNpMXpaWEoyCmFXTmxMVzVsZEhkdmNtc3RjMmxuYm1WeU1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0MKQVFFQXVaWnJ5QUpwMHBJUnNIOHYwL2NFNGFYK05UWVlRL3F1RURXMnVGOVV1RDE4S0JObTE3VUdVYWRvaGxGQgo4TVpGcXh5RVVFM2RvQXpOTk5uK21sYjF0ZGZmTXlyUENLdXVlLyt1TDI4M0VTSGV4UENVaXBEMk5aQ0pNTHhQCjRjRkJOWTI0WHhzeUNnUDRHVzBDTEs2anpORFJVcG1xQXZ1QXRQYUdwd29NbWRIWUViaVlmRCtrUXBlU0VwZTcKNjVXMDRCZGNXOTNSV2NVOFBjcThtajVyeVJ5MzdQWE9TS3Y2Y0QyMHpzM28zc3doSlFqWEo3UzdIUE5pSVBKdQpXQUlEWE1UYm1sRTVvVDdmYXdRVVZFelpBTVR4SHd6Mzg2WCtQbnFpdmJsWlcxMHdDYWV3OWl4a3pnbkJ0eXpSCmpJbmVQdEFPb2ZRK09BWlJqZmpLN3Rid1lRSURBUUFCbzBJd1FEQU9CZ05WSFE4QkFmOEVCQU1DQXFRd0R3WUQKVlIwVEFRSC9CQVV3QXdFQi96QWRCZ05WSFE0RUZnUVVWOVlwZUdOMVpWOWxyVWphc0tOckk4ZGVuc293RFFZSgpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFCV3BydFg5L0JtdC9TRFpNUisrNThPVkRPSGVONUlNNVhtSlRnMUw5WTcxCndaNTJPckFKVGlIeWt5ZjBaZ2RBMVdGczRYQ0tSeWdLNThRMHpoNVRhTjFGYm10L3ppS3M2VGZIcEhyMUVJOU0KOHE1VU4yZW44S0JubmtNNWJhVjRwYnM0MVBqN0xJem11QmR1bGxzVHpqV0t6LytMaUJZaVV0RUlWaXlMckF6UwpxbXJpdGFBZzNyNFpndWFMK0tDV1RSTHBCSmttdXBhb2t3ZG85ckNKdFk1cUg3MEZhb05iZElvZldFNUhpNVJ4CldaRk9xSWhlWkJUUkQvYmJoTFVCQ0lncWFjdndhSVMwTERkSGdQcnBjNUkwTFVGWi9sNFRlOVRzWTRzallRUWYKT0s0bzFhdWQ4SU1ZZWNkZkIxa2hPa2hwRVF6c0JjalRPeUtUUDArenJxMD0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQotLS0tLUJFR0lOIENFUlRJRklDQVRFLS0tLS0KTUlJRGx6Q0NBbitnQXdJQkFnSUlGd1ExS3NCVzR5UXdEUVlKS29aSWh2Y05BUUVMQlFBd1dURlhNRlVHQTFVRQpBd3hPYjNCbGJuTm9hV1owTFd0MVltVXRZWEJwYzJWeWRtVnlMVzl3WlhKaGRHOXlYMnh2WTJGc2FHOXpkQzF5ClpXTnZkbVZ5ZVMxelpYSjJhVzVuTFhOcFoyNWxja0F4TnpJNU56YzFNREV4TUI0WERUSTBNVEF5TkRFek1ETXoKTUZvWERUTTBNVEF5TWpFek1ETXpNVm93V1RGWE1GVUdBMVVFQXd4T2IzQmxibk5vYVdaMExXdDFZbVV0WVhCcApjMlZ5ZG1WeUxXOXdaWEpoZEc5eVgyeHZZMkZzYUc5emRDMXlaV052ZG1WeWVTMXpaWEoyYVc1bkxYTnBaMjVsCmNrQXhOekk1TnpjMU1ERXhNSUlCSWpBTkJna3Foa2lHOXcwQkFRRUZBQU9DQVE4QU1JSUJDZ0tDQVFFQXptbDEKU2pzVUtEWlg3SXQ0eDZQSW1sdTZlZVBobU5kVXhwKzFUR3lVdlUzQktpdi9Kc2J5allCT2czZGlwNThuUEJxOQpyZVRyNHQzWk5uVk1BYm5hV21wd1BSOHNia2Rqa29IRVRpMGlwSVBxRUlRNDdZbjJNSHNZNTRBc1hNY0RTLzNaCmtWU2JKaFJQSmFacTQrWkJaQzBBR2dEZ0FRZDhiS3lYMU8yeWJCcCsvbnAvTW9aQiszTVRmS2E3Ny8yZHFVREwKeThtT0c2VjkvVzdaOTZZdnQrR0pYU2tUS2FTRDhBRGJYSVo3TU03REZsTWkzK3MycGFvb1dncFpJcm9aQjA4UwpybDhMNnIyMm5jVi8zQ2psRG0zS2tvS0hsMXRvMVdkeEtvdDhiOGJTR0MvdHljL3pZQlRjOGxLZWsyQnZkcksrCm1laUMrZ2hWQVNHdEhvd1NYUUlEQVFBQm8yTXdZVEFPQmdOVkhROEJBZjhFQkFNQ0FxUXdEd1lEVlIwVEFRSC8KQkFVd0F3RUIvekFkQmdOVkhRNEVGZ1FVbXN6Ymw0ZUw4Y1A5bXMxZXkrQ2RrVnNQcGVvd0h3WURWUjBqQkJndwpGb0FVbXN6Ymw0ZUw4Y1A5bXMxZXkrQ2RrVnNQcGVvd0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFKMlM2SFRiCjNQeUl2S3NUdmhDeDNZUHNSeVpnTDc2ZmZnM29pWXZ5YytVbk9rbThReTB3d1MySTBieVh5K0tPSVRtYWJKWlEKNk9ka2tKQitoOUNxV3h1V1BobVRtelZsUDYva0p4UktVSnV0YzdReHhidjltUTRNT0doMVFtc1lQYUZ3eWFqbgpmR3o2T2VWcllDR3VuQjk3bTY1c0tCRWNxQVY3RUNCTTUxYVk4RVlVUUxiMHc2N2cvL1lKUGhXbVB1a20wZ1ZhCjVLUTk3a3hsN2R4dXdyVnI3Vk9lWk1hd0hTUVBVY042a0doYWJsTTdNTlJibEMrZGRkaXcrOHVKWDZxM3hiWUgKbVJnb0dEaFhUQjR3ajJiejVaazdzRkd0ODJRb1VZK1RnMng4Q0JwWmtOZ0pBR2lqd3hyWHkzZGtSQW9OSjFyZwpVMUtDTmlsOXVKeVpIUjg9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0=
2024-10-24T13:04:58.930754397Z     server: https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443
2024-10-24T13:04:58.930754397Z   name: ci-op-2fcpj5j6-f6035
2024-10-24T13:04:58.930754397Z contexts:
2024-10-24T13:04:58.930754397Z - context:
2024-10-24T13:04:58.930754397Z     cluster: ci-op-2fcpj5j6-f6035
2024-10-24T13:04:58.930754397Z     user: kubelet
2024-10-24T13:04:58.930754397Z   name: kubelet
2024-10-24T13:04:58.930754397Z current-context: kubelet
2024-10-24T13:04:58.930754397Z preferences: {}
2024-10-24T13:04:58.930754397Z users:
2024-10-24T13:04:58.930754397Z - name: kubelet
2024-10-24T13:04:58.930754397Z   user:
2024-10-24T13:04:58.930754397Z     client-certificate-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUQ4RENDQXRpZ0F3SUJBZ0lJVzBWcUVVNmJnUUl3RFFZSktvWklodmNOQVFFTEJRQXdRakVTTUJBR0ExVUUKQ3hNSmIzQmxibk5vYVdaME1Td3dLZ1lEVlFRREV5TnJkV0psYkdWMExXSnZiM1J6ZEhKaGNDMXJkV0psWTI5dQpabWxuTFhOcFoyNWxjakFlRncweU5ERXdNalF4TWpRNU16SmFGdzB6TkRFd01qSXhNalE1TXpKYU1JRzJNV0F3CkhRWURWUVFLRXhaemVYTjBaVzA2YzJWeWRtbGpaV0ZqWTI5MWJuUnpNRDhHQTFVRUNoTTRjM2x6ZEdWdE9uTmwKY25acFkyVmhZMk52ZFc1MGN6cHZjR1Z1YzJocFpuUXRiV0ZqYUdsdVpTMWpiMjVtYVdjdGIzQmxjbUYwYjNJeApVakJRQmdOVkJBTVRTWE41YzNSbGJUcHpaWEoyYVdObFlXTmpiM1Z1ZERwdmNHVnVjMmhwWm5RdGJXRmphR2x1ClpTMWpiMjVtYVdjdGIzQmxjbUYwYjNJNmJtOWtaUzFpYjI5MGMzUnlZWEJ3WlhJd2dnRWlNQTBHQ1NxR1NJYjMKRFFFQkFRVUFBNElCRHdBd2dnRUtBb0lCQVFDNGFUYW1sRnZzTTF5MEV6WEkyVVpYVEZTanIyZzVJdlFvVlV5aAowdGRodDM2K1NPWkkzeHA2MC9POEw3VFAzMGxGVnpwZ0NVclJqcTNNTDh4VmJ5MnNydHYvc1RVc2hmVDRiTzlaCnE0Ulhud0g1T0RFS3JRK2J3OWdyMDRKUHRqN2phSXJRR0d5YkpYOUFZcVlIWng3bVkxQWpxOUVOSDdzcFlXMU4KQnBmclduRlFjUDhKcEJUR2lDcVVWSm90cG5zcGR6VTdRN3M4a01pWTg0TEQvcDZESzVIR214aTBZcG0rdFhhcgpRRG5XZG5iM3dRVFhDaG5NcWZ0N2R6N0tEaS85VnlGVCtMNFVvblVhQW9YWFJTZ2tTQ0NWSnFjZ0NLcG5sWE54ClZWR1k5elNXcFBtZEsyTlJTaEFBWmMxS2ZPd0pzcXIxOUJOSFI3UExqd0NqSjhVNUFnTUJBQUdqZFRCek1BNEcKQTFVZER3RUIvd1FFQXdJRm9EQVRCZ05WSFNVRUREQUtCZ2dyQmdFRkJRY0RBakFNQmdOVkhSTUJBZjhFQWpBQQpNQjBHQTFVZERnUVdCQlJVQzFQUG1MdWx5YkwzQVgyUzFRQ2Z1SDFid0RBZkJnTlZIU01FR0RBV2dCUlNtTnFqCklDaHZ1ckVEQmJCZ1FwNEZnVWlwNkRBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQVFFQW02ODZ1MGk4VzE5b2NvbGYKUkVDYmsrMWhlVHFxNkpib1RiMGtGbnZ0RTdsN3hFaTFGdkY4MldNU2lrYWVFS05hdUdQZi9LNUtjdHg1TjZWdgo2OWZDQS9nM2cxZDc4Yk85My9rcjRoQUM5TUpjYjVnTGxXQ2t0RmV1bUhhVzRhM0FUMnZPZFhPR3RqOEdTN3dJCjR1VVJKWENkSUFoZllKNThleFlhUWQ5WDBua1d0ekFhMzlGdGJlWm5Jd2kvZXdyWUdwZk5UVFo2YkFhakVsNEoKdGt0SmJWbUt4UHh0T1ZtL1pTVmRBbWVuemhYTHhneDFML3M4SWNvRjArb1FoM09aVUVOeTh3VW5aQ0p0UmE3VAo1QmxESUpKRTM0NFRWK3RuM1dqeEtZb3IwMkhYMXpBdGVhUiszSEJMRE9RTXdlREM4TTBZZWlRYXNabVZlSWoyCnZWelordz09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
2024-10-24T13:04:58.930828848Z     client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBdUdrMnBwUmI3RE5jdEJNMXlObEdWMHhVbzY5b09TTDBLRlZNb2RMWFliZCt2a2ptClNOOGFldFB6dkMrMHo5OUpSVmM2WUFsSzBZNnR6Qy9NVlc4dHJLN2IvN0UxTElYMCtHenZXYXVFVjU4QitUZ3gKQ3EwUG04UFlLOU9DVDdZKzQyaUswQmhzbXlWL1FHS21CMmNlNW1OUUk2dlJEUis3S1dGdFRRYVg2MXB4VUhELwpDYVFVeG9ncWxGU2FMYVo3S1hjMU8wTzdQSkRJbVBPQ3cvNmVneXVSeHBzWXRHS1p2clYycTBBNTFuWjI5OEVFCjF3b1p6S243ZTNjK3lnNHYvVmNoVS9pK0ZLSjFHZ0tGMTBVb0pFZ2dsU2FuSUFpcVo1VnpjVlZSbVBjMGxxVDUKblN0alVVb1FBR1hOU256c0NiS3E5ZlFUUjBlenk0OEFveWZGT1FJREFRQUJBb0lCQUcrYkJYODFSSmlqMFJRTwptRHR3KzdnenN2bkpXNnRRc3pHQnhqb2RyMG90VW1CNUlCMjBJSU43cVFRWVh6WjdMUTN4SjV3ZG81RkY1M3N1CjVseENTQU05WkZuUDlwMTJpUGFjZnhMbTBkWnhuakVsUjlEdFFnZ0VNOXo2U2VCU0xLZFdpc1E4TSthRmdBa0kKYXZFS0FOSndBbjFFSytYa2NWSXU2NFoxV2hZT0E3cEt4NnhPZ0FONnN2SVFwalpFb1ZGMmwrcDJYbnhWQkxTYQpwTmt6Q3BzRlF1ZUFlU2p3L0hmeHZUaEpYcFpHOC9mQ25LUXFtczQ1TmVqQVc4TEJtM0JNVVJQcVdVUmQxMStkClUyUXhzUzF5aUNmb2hRVys3RE1JWjJJSjlzWWFFWEN2b0tINndXY1hJQjlwc2tWMjFWWmdSckRCcEpyM05rMjIKS1VlV2xVMENnWUVBeUw3WlBCN252Y2FWRDNqZCtDTi8rdEFlOXcxZmpaWWtxOW9CMjJydGNBYzRTWEhjaTRqdgpSOWZEc1JKOTJzcEdhUzZPNHlkTkRHSkp6UjB4T3BOVEVlZ3AvWG5UMmM1V0N1SC8rRjNqd25lekJodGVxR0wyCnJsRlBRQ3IwY1dMNjczZGRCSlBlNTlSQXY4Q290NlJYS2lULzhTOEI1NzJldENLY3IzejlrdDhDZ1lFQTZ5dGkKYkRJQ29lV0RyTDBxNVQvblNqUGZBNm5iMVVqOTc1RlNlUkNJbFJiS3JiUjhkRSt0amgyUFhncmh1K0lhQjZtZQpCVGUzdlpvUGdDT1dyR3VzZjBFRndtdjZZQ0xPSDYvb2o1R0dsMktHMnM2UHc5V05WQWMzL2RSdU95d0lwd2U5CjFOZ3NCdnpta1hRL0EyTjNkVXhDMmYyZ0ZMbGFUSGV3c0YyZGd1Y0NnWUFFUFVPSktYM3JYQTQ1eld1aW1kMVMKQnFNZ3hIK0lGZXVnVlhKRWwvVjdzQ3RZSWJkdW11MnlyclBMUVlDMjVoNm1pZjBxZnVINXFQbTJBeS9POU4yRQpPMHJudDNMcDNwbWlrVE0rbzN6ejFtTkpkYnFDQTlVQnlXWExEUjBzWjlLMHkraUMrRHgwMk42bExnUmlIRzVhCnpYMDZNcXZ2b09tQkZGb0t1ckVBTVFLQmdCNytGN21FbWI0MlhqVWN0Uk9EYVhyby9TSWhkNXhUUTB3WVRkTXcKcjU5ZVlwV2o3S2J6ZlQ3MmZGZDdmYVppTk9YL241aTJjQnVudXhVN0czZGJ0VjhMOWJOYXdhTjcwc2h3U1dqcQpWOStKbDVZc2xOdk5RQlByNDZBcko5L2FmTVZ1aXNFTFFEb25pRWR4RjREZDBmZCtkbXorTk9PbVRkVTRET3doCldrblBBb0dCQU1SN3YxbExuL0ZsTTNUdEtnTGI5ZVpGdzB2ZkRpa3ZHd0hVNWZOUzVyQXQ0T05FRWdsNkdjemEKS0JGVzkzUHZML2JneDFLRmdYaDIxUnIwSlR2Qk92V0M3MGgvMGtIS0RrSkxOZWtzS1phQldyQThkUCtIYjVKTQpvRHFrblZzTmhoMTdvbThqakJXb05MQjBEd1F4azd2N1dVRGhkT01mMmFlRUFML09HMkE0Ci0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg==
2024-10-24T13:04:58.938939508Z E1024 13:04:58.938881    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:04:58.957161559Z I1024 13:04:58.954640    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 10664
2024-10-24T13:05:04.731439852Z E1024 13:05:04.731205    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:05:04.748904573Z I1024 13:05:04.748241    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 10858
2024-10-24T13:05:39.579502411Z I1024 13:05:39.579419    7167 daemon.go:872] Starting health listener on 127.0.0.1:8798
2024-10-24T13:08:37.387612394Z W1024 13:08:37.387543    7167 reflector.go:470] k8s.io/client-go/informers/factory.go:160: watch of *v1.Node ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding
2024-10-24T13:08:39.161017657Z W1024 13:08:39.160905    7167 reflector.go:470] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: watch of *v1.ControllerConfig ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding
2024-10-24T13:08:39.161017657Z W1024 13:08:39.160938    7167 reflector.go:470] k8s.io/client-go/informers/factory.go:160: watch of *v1.Node ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding
2024-10-24T13:08:39.161017657Z W1024 13:08:39.160908    7167 reflector.go:470] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: watch of *v1.MachineConfig ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding
2024-10-24T13:08:39.161017657Z W1024 13:08:39.160909    7167 reflector.go:470] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: watch of *v1.ClusterVersion ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding
2024-10-24T13:08:39.161017657Z W1024 13:08:39.160963    7167 reflector.go:470] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: watch of *v1.FeatureGate ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding
2024-10-24T13:08:55.147756876Z W1024 13:08:55.147672    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.147756876Z W1024 13:08:55.147677    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.147838526Z E1024 13:08:55.147752    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.ClusterVersion: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.147838526Z E1024 13:08:55.147771    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.147838526Z E1024 13:08:55.147781    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.ClusterVersion: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.149036876Z E1024 13:08:55.148993    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.659766843Z W1024 13:08:55.659688    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.659940593Z E1024 13:08:55.659903    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:55.659995513Z E1024 13:08:55.659961    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:58.086801006Z W1024 13:08:58.086729    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:58.086869666Z E1024 13:08:58.086795    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:08:58.086869666Z E1024 13:08:58.086826    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:08.280720840Z W1024 13:09:08.280533    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:08.280720840Z E1024 13:09:08.280621    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:08.280720840Z E1024 13:09:08.280658    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:09.998488585Z W1024 13:09:09.998427    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:09.998597805Z E1024 13:09:09.998572    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:09.998676194Z E1024 13:09:09.998656    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:10.491034674Z W1024 13:09:10.490924    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:10.491034674Z E1024 13:09:10.491008    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:10.491143684Z E1024 13:09:10.491053    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:12.937980355Z W1024 13:09:12.937892    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:12.937980355Z E1024 13:09:12.937948    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:12.938052945Z E1024 13:09:12.937975    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:12.961411792Z W1024 13:09:12.961297    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:12.961411792Z E1024 13:09:12.961353    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:12.961411792Z E1024 13:09:12.961385    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:16.701070598Z W1024 13:09:16.700974    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:16.701070598Z E1024 13:09:16.701054    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:16.701178458Z E1024 13:09:16.701086    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:18.242732067Z W1024 13:09:18.242629    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:18.242732067Z E1024 13:09:18.242705    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:18.242801017Z E1024 13:09:18.242743    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:26.888388554Z W1024 13:09:26.888294    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:26.888388554Z E1024 13:09:26.888341    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.ClusterVersion: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:26.888388554Z E1024 13:09:26.888366    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.ClusterVersion: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:27.851818296Z W1024 13:09:27.851723    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:27.851818296Z E1024 13:09:27.851776    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:27.851818296Z E1024 13:09:27.851805    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:28.270525206Z W1024 13:09:28.270396    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:28.270525206Z E1024 13:09:28.270449    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:28.270525206Z E1024 13:09:28.270477    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:31.807044201Z W1024 13:09:31.806958    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:31.807044201Z E1024 13:09:31.807013    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:31.807144241Z E1024 13:09:31.807046    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:32.931397561Z W1024 13:09:32.931310    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:32.931397561Z E1024 13:09:32.931372    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:32.931477931Z E1024 13:09:32.931403    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:40.140459693Z W1024 13:09:40.140363    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:40.140459693Z E1024 13:09:40.140408    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:40.140459693Z E1024 13:09:40.140437    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:42.429114306Z W1024 13:09:42.428999    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:42.429114306Z E1024 13:09:42.429047    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:42.429114306Z E1024 13:09:42.429069    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:45.425519539Z W1024 13:09:45.425419    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:45.425519539Z E1024 13:09:45.425476    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:45.425618189Z E1024 13:09:45.425532    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:56.824338920Z W1024 13:09:56.824225    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:56.824338920Z E1024 13:09:56.824274    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:56.824338920Z E1024 13:09:56.824312    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:09:57.451068440Z W1024 13:09:57.450961    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:57.451068440Z E1024 13:09:57.451008    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:09:57.451068440Z E1024 13:09:57.451031    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:10:02.637184134Z W1024 13:10:02.637071    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:02.637184134Z E1024 13:10:02.637167    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.ClusterVersion: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:02.637260954Z E1024 13:10:02.637200    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.ClusterVersion: failed to list *v1.ClusterVersion: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/clusterversions?resourceVersion=14653": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:10.689918543Z W1024 13:10:10.689803    7167 reflector.go:547] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:10.689918543Z E1024 13:10:10.689860    7167 reflector.go:150] github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:10.689918543Z E1024 13:10:10.689898    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/config/informers/externalversions/factory.go:125: Failed to watch *v1.FeatureGate: failed to list *v1.FeatureGate: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/config.openshift.io/v1/featuregates?resourceVersion=13237": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:12.925576137Z W1024 13:10:12.925439    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:10:12.925576137Z E1024 13:10:12.925490    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:10:12.925576137Z E1024 13:10:12.925544    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:10:13.506248962Z W1024 13:10:13.506147    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:13.506248962Z E1024 13:10:13.506215    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:13.506319602Z E1024 13:10:13.506248    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.MachineConfig: failed to list *v1.MachineConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/machineconfigs?resourceVersion=13210": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:18.266805602Z W1024 13:10:18.266716    7167 reflector.go:547] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:10:18.266805602Z E1024 13:10:18.266766    7167 reflector.go:150] github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:10:18.266805602Z E1024 13:10:18.266792    7167 daemon.go:1402] Got an error from auxiliary tools: github.com/openshift/client-go/machineconfiguration/informers/externalversions/factory.go:125: Failed to watch *v1.ControllerConfig: failed to list *v1.ControllerConfig: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/apis/machineconfiguration.openshift.io/v1/controllerconfigs?resourceVersion=13247": dial tcp 10.0.0.2:6443: connect: connection refused
2024-10-24T13:10:32.237412580Z W1024 13:10:32.237294    7167 reflector.go:547] k8s.io/client-go/informers/factory.go:160: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:32.237412580Z E1024 13:10:32.237379    7167 reflector.go:150] k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:32.237499270Z E1024 13:10:32.237409    7167 daemon.go:1402] Got an error from auxiliary tools: k8s.io/client-go/informers/factory.go:160: Failed to watch *v1.Node: failed to list *v1.Node: Get "https://api-int.ci-op-2fcpj5j6-f6035.XXXXXXXXXXXXXXXXXXXXXX:6443/api/v1/nodes?resourceVersion=14586": dial tcp 10.0.0.2:6443: i/o timeout
2024-10-24T13:10:47.999587668Z I1024 13:10:47.999525    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 10858
2024-10-24T13:13:36.423143262Z I1024 13:13:36.423033    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 10858
2024-10-24T13:15:28.191897613Z E1024 13:15:28.191712    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:15:28.217782002Z I1024 13:15:28.217725    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 19762
2024-10-24T13:15:41.011818059Z E1024 13:15:41.011736    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:15:41.041739627Z I1024 13:15:41.041681    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 20297
2024-10-24T13:15:41.074535496Z E1024 13:15:41.074483    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:15:41.135730622Z I1024 13:15:41.135617    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 20304
2024-10-24T13:15:42.202091728Z E1024 13:15:42.201987    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:15:42.225769207Z I1024 13:15:42.225713    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 20330
2024-10-24T13:19:04.464566393Z I1024 13:19:04.464503    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 20330
2024-10-24T13:21:55.929114617Z E1024 13:21:55.929022    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:21:56.063113419Z I1024 13:21:56.061800    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23198
2024-10-24T13:21:56.113954555Z E1024 13:21:56.113867    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:21:56.179010141Z I1024 13:21:56.178954    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23202
2024-10-24T13:21:56.228266848Z E1024 13:21:56.227489    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:21:56.320180282Z I1024 13:21:56.317463    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23221
2024-10-24T13:21:57.217173463Z E1024 13:21:57.216526    7167 certificate_writer.go:249] /etc/docker/certs.d does not exist yet: open /etc/docker/certs.d: no such file or directory
2024-10-24T13:21:57.326877606Z I1024 13:21:57.325404    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23355
2024-10-24T13:22:03.281893696Z I1024 13:22:03.281797    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23834
2024-10-24T13:22:03.363013040Z I1024 13:22:03.362957    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23838
2024-10-24T13:22:04.891889740Z I1024 13:22:04.891817    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T13:24:39.034481413Z I1024 13:24:39.034378    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T13:30:26.241016966Z I1024 13:30:26.240948    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T13:32:51.561850131Z I1024 13:32:51.561752    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T13:50:51.552103747Z I1024 13:50:51.552009    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T13:54:11.851768478Z I1024 13:54:11.851688    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T14:01:16.154797006Z I1024 14:01:16.154667    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T14:16:27.940467569Z I1024 14:16:27.940343    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 23943
2024-10-24T14:23:33.763843364Z I1024 14:23:33.763766    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 53296
2024-10-24T14:23:33.809500071Z I1024 14:23:33.809411    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 53297
2024-10-24T14:23:34.671301690Z I1024 14:23:34.671171    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 53317
2024-10-24T14:29:28.471338333Z I1024 14:29:28.471264    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 53317
2024-10-24T14:42:28.160384884Z I1024 14:42:28.160307    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59516
2024-10-24T14:42:28.644176453Z I1024 14:42:28.642643    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59529
2024-10-24T14:42:29.037482708Z I1024 14:42:29.037419    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59537
2024-10-24T14:42:31.669178600Z I1024 14:42:31.666223    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59584
2024-10-24T14:42:36.998107400Z I1024 14:42:36.998033    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59699
2024-10-24T14:42:37.063948376Z I1024 14:42:37.063857    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59701
2024-10-24T14:42:37.135977941Z I1024 14:42:37.135841    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59714
2024-10-24T14:42:39.624710562Z I1024 14:42:39.624024    7167 certificate_writer.go:303] Certificate was synced from controllerconfig resourceVersion 59746
